### DISCUSSION TITLE
signalfx.splunk_otel_collector.collector not saving tokens in registry Environment

### DISCUSSION THREAD

**Discussion by Tomasz8603** [signalfx/splunk-otel-collector#4836] [Q&A]
When running the collector role as below, Splunk is getting installed but fails to run because the token will be blank in registry's Environment. I am running it on windows.

- name: "Include splunk_otel_collector"
  include_role:
    name: "signalfx.splunk_otel_collector.collector"
  vars:
    splunk_access_token: "{{ access_token }}"
    splunk_hec_token: "{{ hec_token }}"
    
When I add the tokens manually later I can start the service.

To double check if I do pull the right values I did a test, and the values are fine.
I am pulling the tokens from AWS parameter store so they come in as hashed '***'

test:
- name: copy token to file 
  ansible.windows.win_copy:
      content: "{{ access_token }}"
      dest: C:\temp\tokentest.txt
     
- name: copy token to file 
  ansible.windows.win_copy:
      content: "{{ hec_token }}"
      dest: C:\temp\tokentesthec.txt
     
Ok noticed the parameters are being created with values under 
Computer\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\Environment
but not here:
Computer\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\splunk-otel-collector
It is the second location where I need to stick in the Tokens.
I am using the defaults so using the latest 
Running the gather_facts: true
 "ansible_os_family is Windows"      // I can see that is the condition switch in the ansible role
 
What am I missing?

**Comment by atoulme** [comment]
Thank you for contacting us. We can look at reproducing this issue. Would you please open a support case and post its number here so we can make sure to follow up with you?

**Reply by Tomasz8603** [reply]
ticket: [3482861]
